src/stream/salsa20/salsa20_ivctr64.c

```diff
@@ -15,7 +15,7 @@
 
 #include "tomcrypt.h"
 
-#ifdef LTC_SALSA20
+#if defined(LTC_SALSA20) || defined(LTC_XSALSA20)
```

Comment:
this can be IMO reverted to `#ifdef LTC_SALSA20`

Reply:
:+1:

Reply:
I'd also prefer to add something like the following in the "cleanup" section of `tomcrypt_custom.h` where all the other similar cases are handled

```
#if defined(LTC_XSALSA20) && !defined(LTC_SALSA20)
#error LTC_XSALSA20 requires LTC_SALSA20
#endif
```

then all the changes to `src/stream/salsa20/salsa20_*.c` could be reverted